### PR TITLE
Make pride pins

### PR DESCRIPTION
**Pull request by OCOtheOmega**
# Description
Add pins to indentity pride persons 

# TODO

# Media

# Changelog

:cl:
- add: Added pride pins to loadout

### CODE REVIEW

File: Resources/Prototypes/SimpleStation14/Loadouts/neck.yml [hunk 1]
```diff
@@ -29,3 +29,76 @@
...
+  name: aromantic pin
+  description: "be aro do crime"
+  category: Accessories
+  cost: 1
+  item: ClothingNeckAromanticPin
+
+- type: loadout
+  id: LoadoutNeckAsexualPin
+  name: asexual pin
+  description: "be gay do crime"
```
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Loadouts/neck.yml]: Asexuality is not the same as homosexuality? :P

**Review by Pspritechologist — COMMENTED** [review]
I assume the grammar and lack of punctuation is part of the bit? :P

Besides the one comment I made, this seems good.

**Comment by OCOtheOmega** [comment]
Done

**Review by Pspritechologist — APPROVED** [review]

**Comment by Pspritechologist** [comment]
Edit the pr description to say what it does, please. A changelog entry is fine if you'd like it.

**Comment by Pspritechologist** [comment]
You need to _either_ have an actual Changelog, or no Changelog. You can't leave the nonsense Changelog :P

We have a bot, that'll be put in the game automatically.

**Comment by OCOtheOmega** [comment]
Okay for future PR i will make a more detailed description and changelog
Anyway i write some to make it clear what this PR is about

**Review by DEATHB4DEFEAT — APPROVED** [review]

**Pull request merged**